concepts/05 Widgets/Scheduler/020 Data Binding/16 Web API Service.md

```diff
@@ -51,6 +51,74 @@ DevExtreme provides the <a href="https://github.com/DevExpress/DevExtreme.AspNet
         [dataSource]="store">
     </dx-scheduler>
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :data-source="dataSource"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxScheduler from 'devextreme-vue/scheduler';
+    import * as AspNetData from 'devextreme-aspnet-data-nojquery';
```

Comment:
You can import `createStore` directly since this is all you need in these examples.